hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestCopyOnWriteRollbackActionExecutor.java

```diff
@@ -283,61 +283,66 @@ public void testRollbackScale() throws Exception {
 
   private void performRollbackAndValidate(boolean isUsingMarkers, HoodieWriteConfig cfg, HoodieTable table,
                                           List<FileSlice> firstPartitionCommit2FileSlices,
-                                          List<FileSlice> secondPartitionCommit2FileSlices) throws IOException {
-    //2. rollback
-    HoodieInstant commitInstant;
-    if (isUsingMarkers) {
-      commitInstant = table.getActiveTimeline().getCommitAndReplaceTimeline().filterInflights().lastInstant().get();
-    } else {
-      commitInstant = table.getCompletedCommitTimeline().lastInstant().get();
-    }
+                                          List<FileSlice> secondPartitionCommit2FileSlices) throws IOException, InterruptedException {
+    // Create a client to start timeline service needed by the rollback action executor
+    try (SparkRDDWriteClient client = getHoodieWriteClient(cfg)) {
+      // Sleep for timeline service to start listening on the port
```

Comment:
why it is related with auto commit?

Reply:
The client is closed before this call is made so another client needs to be created here. So creating a separate client here.